src/EnergyPlus/DataSizing.cc

```diff
@@ -314,6 +314,100 @@ void ZoneSizingData::allocateMemberArrays(int const numOfTimeStepInDay)
     this->LatentHeatFlowSeq.dimension(numOfTimeStepInDay, 0.0);
 }
 
+void TermUnitZoneSizingData::allocateMemberArrays(int const numOfTimeStepInDay)
```

Comment:
Some new methods for the new struct `TermUnitZoneSizingData` which is a subset of `ZoneSizingData`.